tensorboard/webapp/metrics/views/main_view/main_view_test.ts

```diff
@@ -484,6 +493,11 @@ describe('metrics main view', () => {
         },
       ]);
 
+      store.overrideSelector(
```

Comment:
One exception from my other request: Can we just move this test case into common_selectors.ts? If its already covered by common_selectors, can we just plain delete this duplicate test case then?